sdk/core/Azure.Core.Expressions.DataFactory/tests/DataFactoryExpressionsTests.cs

```diff
@@ -189,6 +191,19 @@ public void SerailizationOfStringValue()
             Assert.AreEqual(StringJson, actual);
         }
 
+        [Test]
+        public void SerailizationOfListOfT()
+        {
+            var elements = new List<TestModel>
+            {
+                new TestModel { A = 1, B = true },
+                new TestModel { A = 2, B = false }
```

Comment:
can we add a test case for Empty list, null list, and possible a list with one of the items null?